### DISCUSSION TITLE
is there any option to open previous and current month when  numberOfMonths is 2?

### DISCUSSION THREAD

**Discussion by jennifercela** [primefaces/community#744] [PrimeVue]
By default numberOfMonths shows current and next month in dropdown, I have maxDate as today, and it not make sense show next month.
Is there an option or a slot for render previous and current month?
`<Calendar v-model="rangeDates" class="repos" selection-mode="range" :number-of-months="2" :max-date="today" />`
Thanks

**Comment by polinkuer12** [comment]
You can use the show-prev-month attribute on the <Calendar> tag to display the previous month as well. Set this value to true so that the previous and current month are displayed, and set this to false to only display the current and next month. For example:

ruby
Copy the code
<Calendar v-model="rangeDates" class="repos" selection-mode="range" :number-of-months="2" :max-date="today" :show-prev-month="true" / >
After the change, the <Calendar> tag will display the previous month and the current month.  [Quick Draw](https://quikdraw.io)

**Reply by jennifercela** [reply]
Thanks you, but still not work for me
I'm working in "vue": "^3.0.2", with library  "primevue": "^3.39.0",
have you other suggests?

`<Calendar
        v-model="rangeDates"
        class="repos"
        selection-mode="range"
        :max-date="today"
        :manual-input="false"
        :show-icon="true"
        :show-button-bar="true"
        :show-prev-month="true"
        :number-of-months="2"
        :required="true"
        date-format="dd/mm/yy"
        @hide="updateValueCalendar"
      />`

**Comment by cetincakiroglu** [comment]
Hi,

This topic belongs to PrimeVue. I'm changing the category.

**Reply by thomasbalsloev** [reply]
I face the same issue.
I would like our datepicker to show the current and previous month because it is used so select data in the past.
